test/unit/lib/redmine/menu_manager/menu_helper_test.rb

```diff
@@ -100,14 +100,14 @@ def test_render_menu_node_with_nested_items
 
     html_node = HTML::Document.new(@response.body)
     assert_select(html_node.root, "li") do
-      assert_select("a.parent-node", "Parent node")
+      assert_select("a.parent-node-menu-item", "Parent node")
       assert_select("ul") do
-        assert_select("li a.child-one-node", "Child one node")
-        assert_select("li a.child-two-node", "Child two node")
+        assert_select("li a.child-one-node-menu-item", "Child one node")
+        assert_select("li a.child-two-node-menu-item", "Child two node")
         assert_select("li") do
-          assert_select("a.child-three-node", "Child three node")
+          assert_select("a.child-three-node-menu-item", "Child three node")
           assert_select("ul") do
-            assert_select("li a.child-three-inner-node", "Child three inner node")
+            assert_select("li a.child-three-inner-node-menu-item", "Child three inner node")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.